src/metabase/models/recent_views.clj

```diff
@@ -155,80 +162,110 @@
 (defmulti fill-recent-view-info
   "Fills in additional information for a recent view, such as the display name of the object.
 
-  - When called from `GET /popular_items`, the `model_object` field will be present, and should be used instead of
-  querying the database for the object."
+  For most things, we try to gather the information in a single query, but for cards and datasets, but certain things
+  are way easier to check with code (e.g. a collection's parent collection.)"
```

Comment:
I think the wording/grammar here is off?

Reply:
Fixed